readme.md

```diff
@@ -167,6 +167,7 @@
 - [shellChatGPT](https://github.com/mountaineerbr/shellChatGPT) - Use ChatGPT from the command-line.
 - [Shell Genie](https://github.com/dylanjcastillo/shell-genie) - Interact with the terminal in plain English.
 - [ShellGPT](https://github.com/TheR1D/shell_gpt) - Use ChatGPT from the command-line.
+- [ChatGPT-shell-cli](https://github.com/0xacx/chatGPT-shell-cli) - Use ChatGPT from the command-line. No Python or JS required. 
```

Comment:
> No Python or JS required. 

This is not the selling point you think it is. Shell scripts are completely unmaintainable, so this just makes me believe it has more bugs or is less user-friendly.

Reply:
Thank you for merging this! 🙏 
I definitely agree that shell scripts are quite hard to maintain, but it is very cool to be able to access chatgpt from a remote server, a docker container or a raspberry pi without installing the whole Python or JS ecosystem. The script has a few known bugs but in general works very well. Please feel free to try it, I would love to hear your feedback!